app/src/androidTest/java/net/skyscanner/backpack/compose/select/BpkSelectTest.kt

```diff
@@ -0,0 +1,34 @@
+/**
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.skyscanner.backpack.compose.select
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import net.skyscanner.backpack.compose.BpkSnapshotTest
+import net.skyscanner.backpack.demo.compose.SelectStory
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class BpkSelectTest : BpkSnapshotTest() {
+
+    @Test
+    fun default() = snap {
```

Comment:
we generally try to have separate tests for each state, rather than the full story to ensure screenshot failures will highlight what exactly failed.
the error state with text should be part of the `field set` tests as described further down
we should also have a test covering the expanded state. you can check the `BpkBarChartTest` selected test on how to use espresso assertions - i believe you should be able to utilise that with providing a click action

Reply:
ok